templates/statefulset-zabbix-proxy.yaml

```diff
@@ -147,4 +147,6 @@ spec:
       volumes:
       {{- toYaml . | nindent 8 }}
       {{- end }}
+  volumeClaimTemplates:
```

Comment:
Hi @tomsozolins!

Good job here. Do you can change this block to?

```
  {{- with .Values.zabbixproxy.extraVolumeMounts }}
  volumeClaimTemplates:
    {{- toYaml .Values.zabbixproxy.extraVolumeClaimTemplate | nindent 4 }}
  {{- end }}
```

Reply:
extraVolumeMounts is under different template section, this wouldn't work?

Reply:
Sorry @tomsozolins, error of copy/past. I fix the suggestion.

Reply:
@aeciopires adjusted

Reply:
Hi @tomsozolins! Do you make the commit? I don't see the changes in this file.